Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ProjectController.cs

```diff
@@ -250,22 +229,21 @@ public class ProjectController : Controller
         * @apiSuccessExample {json} Success-Response:
         * HTTP/1.1 200 OK 
         * {
-        *     "description":"Object of ProjectAc"
+        *     "description":"Method to return list of users/employees of the given group name"
         * }
         */
         [ServiceFilter(typeof(CustomAttribute))]
         [HttpGet]
         [Route("fetchProject/{name}")]
-        public ProjectAc Fetch(string name)
+        public IActionResult Fetch(string name)
```

Comment:
Purpose of this method to fetch the users by group name. 
updated this method now i have use asyn and given proper name.